examples/chat-online-status/javascript/index.html

```diff
@@ -9,7 +9,7 @@
     <title>Online Status Demo</title>
   </head>
   <body class="font-inter">
-    <div class="max-w-md mx-auto p-6 bg-white rounded-lg shadow-lg">
+    <div class="max-w-md mx-auto p-6 bg-white rounded-lg shadow-lg text-N1000">
```

Comment:
Is text-N1000 supplied by franken ui? It's not an ably-ui colour, I just wanted to check that this was intentional

Reply:
No, you're right, I assumed it had. I've added a class for this now.